### PR TITLE
feat: [#1415] move prefix to url

### PR DESCRIPTION
**Pull request by maria-j-k**
- prefix renamed to scope
- scope is send in url params
- global filters has been commented out (temporary fix -excessive calls to `FiltersConfig` from collections navbar) 

closing #1415

### CODE REVIEW

**Comment by Marcelinna** [comment]
When I select the filter other disappear 
image

**Comment by Marcelinna** [comment]
PR I dont see community catalog, search result
image

PR-12
image

**Comment by Marcelinna** [comment]
I cant click on Browse EU Resources tab

**Comment by Michal-Kolomanski** [comment]
@maria-j-k I still see `COLLECTIONS_PREFIX` value in the local storage. Its not changing so I assume that this is a default value. Please remove it 
image

File: ui/apps/ui/src/app/collections/services/custom-route.service.ts [hunk 1]
```diff
@@ -23,8 +23,11 @@ import {
 } from '@collections/repositories/types';
 import { FilterService } from '@components/filters/filters.service';
 
+export const DEFAULT_SCOPE = 'eu';
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/collections/services/custom-route.service.ts:26]: default scope should be set to `"pl"`

File: ui/apps/ui/src/app/collections/services/custom-route.service.ts [hunk 2]
```diff
@@ -130,9 +137,9 @@ export class CustomRoute {
   _updateParamsBy(collection: string, currentUrl: string): ICustomRouteProps {
     const parsedQueryParams = queryParamsMapFrom(currentUrl);
     const q = (parsedQueryParams['q'] as string | undefined) ?? '*';
-
+    const scope = (parsedQueryParams['scope'] as string | undefined) ?? 'eu';
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/collections/services/custom-route.service.ts:140]: "pl"

**Review by Marcelinna — APPROVED** [review]

**Comment by maria-j-k** [comment]
> @maria-j-k I still see `COLLECTIONS_PREFIX` value in the local storage. Its not changing so I assume that this is a default value. Please remove it image

@Michal-Kolomanski 
this prefix is stored in the browser storage from previous run. If you remove it it won't be saved again

**Comment by Marcelinna** [comment]
@Michal-Kolomanski @Marcelinna 
it is ok when I enter https://search-2.docker-fid.grid.cyf-kr.edu.pl/
I go to https://search-2.docker-fid.grid.cyf-kr.edu.pl/search/all_collection?q=* 
Url is without scope

**Comment by Marcelinna** [comment]
And it is only pr problem?
When I click on service name 

I am redirected to https://beta.marketplace.eosc-portal.eu/services/ai4eosc-platform/information

on pr-12 
I am redirected to https://marketplace-12.docker-fid.grid.cyf-kr.edu.pl//services/ai4eosc-platform/offers

**Comment by Michal-Kolomanski** [comment]
> > @maria-j-k I still see `COLLECTIONS_PREFIX` value in the local storage. Its not changing so I assume that this is a default value. Please remove it image
> 
> @Michal-Kolomanski this prefix is stored in the browser storage from previous run. If you remove it it won't be saved again

correct

**Comment by Michal-Kolomanski** [comment]
> And it is only pr problem? When I click on service name image I am redirected to https://beta.marketplace.eosc-portal.eu/services/ai4eosc-platform/information
> 
> on pr-12 I am redirected to https://marketplace-12.docker-fid.grid.cyf-kr.edu.pl//services/ai4eosc-platform/offers

expected behaviour. PRs are connected to beta still

**Review by Michal-Kolomanski — APPROVED** [review]

**Pull request merged**